packages/transport/src/modules/accounts/AccountController.ts

```diff
@@ -404,6 +433,7 @@ export class AccountController {
             this.info.set("device", device.toJSON()),
             this.info.set("identity", deviceSharedSecret.identity.toJSON()),
             this.info.set("baseKey", privBaseDevice.toJSON()),
+            this.info.set("isHardwareBased", isHardwareBased),
```

Comment:
I would rather go with setting `baseKey` for legacy and `baseKeyHandle` with the CAL. Or we should rename `isHardwareBased` to `isBaseKeyStoredOnHardware`.